README.md

```diff
@@ -37,7 +37,7 @@ To make it work also for debug builds add this to your `Cargo.toml`
 opt-level = 3
 ```
 
-Make sure to call the functions in an interrupt-free context.
+Make sure to call the functions in an interrupt-free context. On ESP32 there is the default-feauture `critical-section` which will ensure this.
```

Comment:
I suppose the fear is that if an interrupt fires and tries to execute code that's not in the CPU cache, the cache will try and do a read from flash. Please correct me if I'm wrong!

If what I mentioned is the case, should we be using critical-sections by default for all chips then?